target/linux/ramips/mt7620/base-files/etc/board.d/01_leds

```diff
@@ -41,8 +41,9 @@ bdcom,wap2100-sk)
 	ucidef_set_led_netdev "wifi_led" "wifi" "green:wlan2g" "wlan0"
 	;;
 bolt,bl201)
-	ucidef_set_led_netdev "wlan0" "wlan0-link" "blue:wlan" "wlan0"
-	ucidef_set_led_netdev "wan" "eth0.2-link" "blue:wan" "eth0.2" "link"
+	ucidef_set_led_netdev "phy0-ap0" "phy0-ap0" "blue:wlan" "phy0-ap0"
+	ucidef_set_led_netdev "wan" "eth0.2" "blue:wan" "eth0.2" 
+	ucidef_set_led_netdev "lan" "eth0.1" "blue:wps" "eth0.1" 
```

Comment:
Why assign LAN activity to WPS LED?

Reply:
Since this board had no LAN LED, and WPS is not really used, so I think it's a good idea to use WPS LED as LAN LED.
Should I just remove it?

Reply:
yes, please remove it. it would confuse users.

Reply:
resolved, and squashed my commit into one.